jobsAdmin/JobsAdmin.go

```diff
@@ -508,53 +508,62 @@ func (ja *jobsAdmin) TryGetPerformanceAdvice(bytesInJob uint64, filesInJob uint3
 	return a.GetAdvice()
 }
 
-
-//Structs for messageHandler
-
-/* PerfAdjustment message. */
-type jaPerfAdjustmentMsg struct {
-	Throughput int64 `json:"cap-mbps,string"`
-}
-
-func (ja *jobsAdmin) messageHandler(inputChan <-chan common.LCMMsg) {
+func (ja *jobsAdmin) messageHandler(inputChan <-chan *common.LCMMsg) {
 	toBitsPerSec := func(megaBitsPerSec int64) int64 {
 		return megaBitsPerSec * 1000 * 1000 / 8
 	}
-	glcm := common.GetLifecycleMgr()
 	
 	lastPerfAdjustTime := time.Now() // right when this routine songs
```

Comment:
Consider adjusting lastPerfAdjustTime to time.Now - 2 * minIntervalBetweenPerfAdjustment or something like that